src/lib/InstantSearch.js

```diff
@@ -339,7 +339,12 @@ class InstantSearch extends EventEmitter {
    */
   dispose() {
     this.removeWidgets(this.widgets);
+    // allow to restart this instance
```

Comment:
```suggestion
    // You can not start an instance two times, therefore a disposed instance needs to set started as false
    // otherwise this can not be restarted at a later point.
```